pipeline/test/update-prismic-snapshots.ts

```diff
@@ -206,6 +206,14 @@ const addCommentsToUpdateScript = async (docs: PrismicDocument[]) => {
 };
 const main = async () => {
   const client = createPrismicClient();
+
+  // delete all first
+  fs.readdir(dataDir).then(result => {
+    result
+      .filter(f => /[.]json$/.test(f))
+      .map(f => fs.unlink(`${dataDir}/${f}`));
+  });
```

Comment:
I'd added one and then chose to remove it, but its JSON file wasn't removed. So I think this needed to be added - delete all files before recreating the listed ones.